test/radius_SUITE.erl

```diff
@@ -315,6 +316,48 @@ vendor_dicts(Config) ->
     application:set_env(ergw_aaa, apps, OrigApps),
     ok.
 
+vendor_dicts_02() ->
+    [{doc, "Vendor Dicts"}].
+vendor_dicts_02(Config) ->
```

Comment:
Perhaps `vendor_dicts_ituma`() is a better function name?

Reply:
Mmmm, both are Ituma actually :-/ I'll check a better name, yes.

Reply:
vendor_dicts_ituma_gps_location vs vendor_dicts_ituma_str_location?

Reply:
If there are only these two for Ituma they can be differentiated by
GPS and nothing.
vendor_dicts_ituma_gps_location vs vendor_dicts_ituma_location
This is if I have understood the difference correctly.
Ie, location is always a string. Either GPS format, or not.

Reply:
this not a different dictionary, only a different way to fill ter values.

Reply:
Hmmm, the end result is a string, but it may be built from two GPS formatted values (gps lat and long), or just copied from whatever value we supply (the new addition).

Note that the test radius server will check in both cases that an IM_LI_Location AVP is present in the request (although built from different session attributes)

Reply:
But the names are fine for me :)